hack/create-operatorhub-prs.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+# Copyright 2023 Red Hat, Inc. and/or its affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+VERSION=$1
+if [[ $1 == v* ]]; then TAG=$1; else TAG=v$1; fi
+GITHUB_AUTHOR=$2
+COMMUNITY_OPERATORS=community-operators
+COMMUNITY_OPERATORS_PROD=community-operators-prod
+if [[ $3 == false ]]; then DRY_RUN=false; else DRY_RUN=true; fi
+if [ -z "$4" ]; then KOGITO_OPERATOR_DIR=$(pwd); else KOGITO_OPERATOR_DIR=$4; fi
+
+echo "Kogito Operator directory is ${KOGITO_OPERATOR_DIR}"
+
+git remote add upstream https://github.com/kiegroup/kogito-operator.git >/dev/null 2>&1
+git fetch upstream
+git fetch upstream --tags
+echo "Checking out Kogito $TAG"
+git checkout -B kogito-$TAG $TAG
+
+cd /tmp
+
+create_operatorhub_pr() {
+  echo "### Starting changes on $1 repo ####"
+  if [ -d "$1" ];
+  then
+      echo "$1 directory exists."
+      cd $1
+      git fetch --all
```

Comment:
don't need to fetch all, just the tag we want to use.

Reply:
So we need only the the main?

Reply:
Using a `git pull --rebase upstream main` should be the best option probably, wdyt?

Reply:
for which repository?
for Kogito you should pull 1.3x while the operator hub ones should be main.

Reply:
ok, I got here, there is no need to do git pull rebase, it is better to always begin clean, delete everything and clone again, a clean git clone will land in the main branch, so no need to care about it.

As @radtriste suggested, use a temporary directory for this.

Reply:
As discussed, now I'm using a new fresh tmp folder via `mktemp` command.